daq/network.py

```diff
@@ -100,7 +100,12 @@ def add_host(self, name, cls=DAQHost, ip_addr=None, env_vars=None, vol_maps=None
         params['tmpdir'] = os.path.join(tmpdir, 'nodes') if tmpdir else None
         params['env_vars'] = env_vars if env_vars else []
         params['vol_maps'] = vol_maps if vol_maps else []
-        host = self.net.addHost(name, cls, **params)
+        try:
+            host = self._retry_func(partial(self.net.addHost, name, cls, **params))
+        except Exception as e:
+            # Need to free any ip allocations.
```

Comment:
Preface with TODO

Reply:
This comment meant what the following line was doing already.